.gitignore

```diff
@@ -1,5 +1,7 @@
 twistd.log*
 twistd.pid
 *.pyc
+ex-*.py
```

Comment:
What's this ignore for?

Reply:
I did write some code for your exercises at the end of each lesson. Those programs started with ex-. At this point, I could just delete those files and revert the .gitignore.

Reply:
I see, the `ex` stands for "exercise". That makes sense now. Why don't we add a comment here explaining the purpose.